typings/numba.pyi

```diff
@@ -39,5 +39,5 @@ def jit(
     parallel: bool = ...,
 ) -> Callable[[F], F]: ...
 
-njit = jit
-generated_jit = jit
+njit = ...
+generated_jit = ...
```

Comment:
I'm quite sure we want to keep those? can they be a TypeAlias?

Reply:
Sure they can. It seems that TypeAlias is imported from the wrong library. It should be imported from <code>typing_extensions</code>.